### PR TITLE
Graph doc

### PR DESCRIPTION
**Pull request by yiheng**
## What changes were proposed in this pull request?
* Add Input layer to python and document
* Replace Graph[Float] with tensor numeric

## How was this patch tested?
PR validation

## Related links or issues (optional)
https://github.com/intel-analytics/BigDL/issues/1261

### CODE REVIEW

**Comment by yiheng** [comment]
PR validation pass

File: docs/docs/APIdocs/Layers/Containers/Graph.md [hunk 1]
```diff
@@ -34,17 +34,22 @@ model = Model([Node], [Node])
...
+val input1 = Input()
+val input2 = Input()
+val cadd = CAddTable().inputs(input1, input2)
+val graph = Graph(Array(input1, input2), cadd)
+
+val output = graph.forward(T(Tensor(T(0.1f, 0.2f, -0.3f, -0.4f)),
+                             Tensor(T(0.5f, 0.4f, -0.2f, -0.1f))))
+val gradInput = graph.backward(T(Tensor(T(0.1f, 0.2f, -0.3f, -0.4f)),
+                                 Tensor(T(0.5f, 0.4f, -0.2f, -0.1f))),
+							   Tensor(T(0.1f, 0.2f, 0.3f, 0.4f)))
```
Comment (jason-dai) [inline docs/docs/APIdocs/Layers/Containers/Graph.md]: Ident

File: docs/docs/APIdocs/Layers/Utilities/Input.md [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+**Scala:**
+```scala
+val input = Input()
+```
+**Python:**
+```python
+input = Input()
+```
+
+Input layer do nothing to the input tensors, just pass them. It should be used as input node
```
Comment (jason-dai) [inline docs/docs/APIdocs/Layers/Utilities/Input.md]: just pass them through

File: docs/docs/APIdocs/Layers/Containers/Graph.md [hunk 3]
```diff
@@ -34,17 +34,22 @@ model = Model([Node], [Node])
```
Comment (jason-dai) [inline docs/docs/APIdocs/Layers/Containers/Graph.md]: You should add reference link to Input

File: docs/docs/APIdocs/Layers/Utilities/Input.md (hunk 2 above)
Comment (jason-dai) [inline docs/docs/APIdocs/Layers/Utilities/Input.md]: It is used as input to the Graph container (add a link) when the first layer of the graph container ...

File: pyspark/bigdl/nn/layer.py [hunk 4]
```diff
@@ -1630,6 +1630,26 @@ def __init__(self,
                  bigdl_type="float"):
         super(CosineDistance, self).__init__(None, bigdl_type)
 
+class Input(Node):
+
+    '''
+    Input layer do nothing to the input tensors, just pass them. It should be used as input node
+    when the first layer of your module accepts multiple tensors as inputs.
```
Comment (jason-dai) [inline pyspark/bigdl/nn/layer.py]: See comments above

File: docs/docs/APIdocs/Layers/Containers/Graph.md (hunk 3 above)
Comment (yiheng) [inline docs/docs/APIdocs/Layers/Containers/Graph.md]: added

**Pull request merged**